xla/service/gpu/model/hlo_op_profiles.h

```diff
@@ -50,22 +50,21 @@ class HloOpProfiles {
 
   // Loads profiles from the given text proto data.
   static std::unique_ptr<HloOpProfiles> Load(
-      absl::string_view profiles_text_proto,
-      absl::string_view default_profile_name);
+      absl::string_view profiles_text_proto);
 
   const HloOpProfile& GetProfile(
       const se::DeviceDescription& device_info) const;
 
-  const HloOpProfile& GetDefaultProfile() const { return default_profile_; }
+  const HloOpProfile& GetLatestProfile() const { return latest_profile_; }
 
  private:
-  HloOpProfiles(ProfilesNestedMap profiles,
-                absl::string_view default_profile_name)
-      : profiles_(std::move(profiles)),
-        default_profile_(profiles_.at(default_profile_name)) {}
+  HloOpProfiles(ProfilesNestedMap profiles)
```

Comment:
Could you mark this constructor `explicit`?  
(getting a linter warning for that)

Reply:
done